### PR TITLE
update for halogen1.0.0

### PR DESCRIPTION
**Pull request by cryogenian**
Depends on slamdata/purescript-halogen-css#15

@garyb please review

### CODE REVIEW

File: example/src/Main.purs [hunk 1]
```diff
@@ -65,12 +67,10 @@ render state =
   where
   renderOne ix =
     HH.div
-      [ HP.key ("echarts-" <> show ix) ]
+      [  ]
```
Comment (garyb) [inline example/src/Main.purs]: Minorest of nitpicks: could use `div_` here :smile:

File: src/Halogen/ECharts.purs [hunk 2]
```diff
@@ -31,65 +32,72 @@ import ECharts.Monad as EM
...
+  . (MonadAff (EChartsEffects eff) g)
+  ⇒ { width ∷ Int, height ∷ Int | r}
+  → H.Component HH.HTML EChartsQuery Unit EChartsMessage g
+echarts = echarts' $ const Nothing
+
+echarts'
+  ∷ ∀ eff g i r
+  . (MonadAff (EChartsEffects eff) g)
+  ⇒ (i → Maybe (EChartsQuery Unit))
+  → { width ∷ Int, height ∷ Int | r }
```
Comment (garyb) [inline src/Halogen/ECharts.purs]: Maybe we should make the dimensions the input value for the component? Seems like a prime candidate for where that kind of thing would be great. The parameterised input thing here is pretty cool, but I think dimensions is by far the most useful possible thing we could use `i` for (it was actually the motivating case for why I added inputs to this release) - manual querying seems more appropriate for updating with new data, etc.

I guess we could do a mixture of the two, if we used `Tuple Dimensions i -> ...` perhaps?

**Review by garyb — COMMENTED** [review]
Would like to see what you think about that dimensions thing before merging. The rest looks good though!

I think the `package.json` will need updating with an `--` for the psa arguments too, Travis is failing before that even due to the halogen dependency mismatch.

File: src/Halogen/ECharts.purs (hunk 2 above)
Comment (cryogenian) [reply src/Halogen/ECharts.purs]: :+1:

**Comment by cryogenian** [comment]
+ Removed redundant `[ ]`
+ Changed input type from `i` to `Dimensions /\ i`

**Review by garyb — APPROVED** [review]

**Pull request merged**